sorc/ncep_post.fd/MISCLN.f

```diff
@@ -839,27 +836,10 @@ SUBROUTINE MISCLN
             if(LVLS(IFD,IGET(587))>0) ITYPEFDLVL(IFD)=2
            ENDIF
 
-	   IF (IGET(601)>0) THEN
-            IF (LVLS(IFD,IGET(601))>1) ITYPEFDLVL(IFD)=2
-           ENDIF
-	   IF (IGET(602)>0) THEN
-            IF (LVLS(IFD,IGET(602))>1) ITYPEFDLVL(IFD)=2
-           ENDIF
-	   IF (IGET(603)>0) THEN
-            IF (LVLS(IFD,IGET(603))>1) ITYPEFDLVL(IFD)=2
-           ENDIF
-	   IF (IGET(604)>0) THEN
-            IF (LVLS(IFD,IGET(604))>1) ITYPEFDLVL(IFD)=2
-           ENDIF
-	   IF (IGET(605)>0) THEN
-            IF (LVLS(IFD,IGET(605))>1) ITYPEFDLVL(IFD)=2
-           ENDIF
-
          ENDDO
 !         print *,'call FDLVL with ITYPEFDLVL: ', ITYPEFDLVL,'for tmp,lvls=',LVLS(1:15,iget(59)), &
 !          'grib2tmp lvs=',LVLS(1:15,iget(586))
 
-         CALL FDLVL(ITYPEFDLVL,T7D,Q7D,U7D,V6D,P7D,ICINGFD,AERFD)
```

Comment:
@zhanglikate This calling is needed for calculating other flight level fields. You may remove "AERFD" in this statement.

Reply:
@WenMeng-NOAA  Updated.